diff --git a/configure b/configure
index 090a938136..5104aa5966 100755
--- a/configure
+++ b/configure
@@ -5551,8 +5551,10 @@ enabled libx264           && { use_pkg_config x264 "stdint.h x264.h" x264_encode
                                die "ERROR: libx264 must be installed and version must be >= 0.118."; } &&
                              { check_cpp_condition x264.h "X264_MPEG2" &&
                                enable libx262; }
-enabled libx265           && require_pkg_config x265 x265.h x265_api_get &&
-                             { check_cpp_condition x265.h "X265_BUILD >= 68" ||
+enabled libx265           && { use_pkg_config x265 x265.h x265_api_get ||
+                               { require libx265 x265.h x265_version_str -lx265 &&
+                                 warn "using libx265 without pkg-config"; } } && 
+                            { check_cpp_condition x265.h "X265_BUILD >= 68" ||
                                die "ERROR: libx265 version must be >= 68."; }
 enabled libxavs           && require libxavs xavs.h xavs_encoder_encode -lxavs
 enabled libxvid           && require libxvid xvid.h xvid_global -lxvidcore
