diff --git a/libtiff/CMakeLists.txt b/libtiff/CMakeLists.txt
index 087dfa9e..1d72c1a9 100644
--- a/libtiff/CMakeLists.txt
+++ b/libtiff/CMakeLists.txt
@@ -125,6 +125,9 @@ if(NOT CYGWIN)
     # seems to be the default.
     set_target_properties(tiff PROPERTIES VERSION ${SO_VERSION})
 endif()
+if(NOT ${BUILD_SHARED_LIBS} AND ${CMAKE_CXX_COMPILER_ID} MATCHES "MSVC" )
+	target_compile_options(tiff PUBLIC "/MT")
+endif()
 if(HAVE_LD_VERSION_SCRIPT)
   set_target_properties(tiff PROPERTIES LINK_FLAGS
                         "-Wl,--version-script=${CMAKE_CURRENT_SOURCE_DIR}/libtiff.map")
